release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -93,6 +92,15 @@ Previously, {op-system} DHCP kernel parameters were not working as expected beca
 
 {op-system} now supports multipath on the primary disk, allowing stronger resilience to hardware failure so that you can set up {op-system} on top of multipath to achieve higher host availability. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1886229[*BZ#1886229*] for more information.
 
+[IMPORTANT]
+====
+Only enable multipathing with kernel arguments within a machine config as documented. Do not enable multipathing during installation.
+
+//Update the following xref using the new .adoc created by fix for https://bugzilla.redhat.com/show_bug.cgi?id=1942192
+
+//For more information, see xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installation-user-infra-machines-iso-ibm-z_installing-ibm-z[Creating Red Hat Enterprise Linux CoreOS (RHCOS) machines].
```

Comment:
We can't use a direct link to the specific module (recently merged [here](https://github.com/openshift/openshift-docs/pull/31140/files#diff-d4372e61cf107136f64c7c5c728625482d15812b3a92512e2560e3ac94cc1f4aR5-R6)), so you could say something like:

`For more information, see "Enabling multipath with kernel arguments" in xref:../post_installation_configuration/machine-configuration-tasks.adoc#rhcos-enabling-multipath_post-install-machine-configuration-tasks [Post-installation machine configuration tasks].`

Reply:
Thanks, added!